src/ieee80211_station.erl

```diff
@@ -1733,3 +1733,49 @@ handle_session_timer_ev({_, Level, _} = Ev, {Interval, _, _Opts} = Timer,
 		Data0
 	end,
     {keep_state, Data}.
+
+add_location(AC, Map) when is_binary(AC) ->
+    ?LOG(debug, "Getting location through: ~p", [AC]),
+    try capwap_ac:get_info(AC) of
+	{ok, #{id := Name}} ->
+                ?LOG(debug, "Getting location for name: ~p", [Name]),
+	        AVPName = capwap_config:get(ac, location_avp, undefined),
+	        case capwap_loc_provider:get_loc(Name, false) of
+	            {location, LatVal, LongVal} ->
+	                ?LOG(debug, "Successful location: ~p", [{LatVal, LongVal}]),
+	                Map#{AVPName => iolist_to_binary(io_lib:format("Lat:~s;Lon:~s", [LatVal, LongVal]))};
+	            {error, Cause} ->
+	                ?LOG(error, "Error retrieving location: ~p", [Cause]),
+	                Map
+	        end
+    catch
+	Class:Error:ST ->
+	    ?LOG(warning, "Getting location failed: ~p:~p with ~0p", [Class, Error, ST]),
+	    Map
+	end;
+% FIXME Atoms are allowed for test cases. Test cases should, however,
+% use DTLS and not plain UDP.
+add_location(ACAtom, Map) when is_atom(ACAtom) ->
+    ?LOG(debug, "Getting location through: ~p", [ACAtom]),
+    AC = atom_to_binary(ACAtom),
```

Comment:
missed one:
> src/ieee80211_station.erl:1760: function atom_to_binary/1 undefined

Reply:
Yes, caught and corrected. CI seems ok now.